test/integration/resource_filter.js

```diff
@@ -632,21 +666,21 @@ describe("Resource filter", () => {
   });
 
   describe("descriminated schemas", () => {
-    // we need the accountFilter to be defined since its creation adds
-    // an entry in resource_filter's excludedMap
```

Comment:
This no longer applies since we instanciate a single filter and add models to it.